.gitignore

```diff
@@ -87,4 +87,3 @@ svgReact/svgs
 # Build
 *.map
 build.mjs
-src/styles/constants
```

Comment:
I think the reason I put this into `.gitignore` because it's generated by the script and only used for Amino consumer (since Amino can use directly constant from `build-utils` folder). Do we also want to use this constant file inside of Amino ? 🤔

Reply:
@Thienhuynh95 and I chatted about this and determined we could add a comment to the `theme.ts` file which will keep developers from accidentally editing it.